projects/example-lumen/api.R

```diff
@@ -3,6 +3,16 @@ library(httr)
 library(jsonlite)
 library(readr)
 
+
+lumen_dataset_url<- function(lumen_instance, dataset_id){
+     lumen_domain <- "akvotest"
+     if(Sys.getenv("K8S_ENVIRONMENT") == "production"){
```

Comment:
Add an explicit LUMEN_DOMAIN config

Doing logic based on the environment always ends up in a big mess.

Reply:
d6a3ca9 an attempt to fix that